Apps/WebClient/src/ClientApp/src/store/modules/config/getters.ts

```diff
@@ -22,4 +23,26 @@ export const getters: GetterTree<ConfigState, RootState> = {
         const { webClient } = config;
         return webClient;
     },
+    isOffline(state: ConfigState): boolean {
+        const webclientConfig = state.config.webClient;
+        const clientIP = webclientConfig.clientIP || "";
+        const offlineConfig = webclientConfig.offlineMode;
+
+        if (offlineConfig) {
+            const startTime = new DateWrapper(offlineConfig.startDateTime);
+            const endTime = offlineConfig.endDateTime
+                ? new DateWrapper(offlineConfig.endDateTime)
+                : DateWrapper.fromNumerical(2050, 12, 31);
+
+            const now = new DateWrapper();
+            if (
+                now.isAfterOrSame(startTime) &&
+                now.isBeforeOrSame(endTime) &&
+                !offlineConfig.whitelist.includes(clientIP)
```

Comment:
@sslaws here is the whitelist check

Reply:
right, this has passed both of my local and cypress' tests